angular-workspace/projects/ni/nimble-angular/table-column/enum-text/nimble-mapping-text.directive.ts

```diff
@@ -0,0 +1,32 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
```

Comment:
Keep consistency with nimble components' organization.

```
/mapping/text
/table-column/enum
```

So add the extra entry points and modules.

If the change in consistency was made to save a couple module import lines that's not worth it. Eventually we will move to single file components so the extra module import won't happen.

Reply:
Re-opening. Can you move the mapping elements to secondary entrypoints to align with the other table related elements. That was the description the paths above were illustrating along with the description "add the extra entry points and modules" (was trying to think ahead :P)

Reply:
Whoops, read too fast and missed that aspect.